specification/monitor/resource-manager/readme.md

```diff
@@ -34,7 +34,38 @@ title: MonitorManagementClient
 ``` yaml
 description: Monitor Management Client
 openapi-type: arm
-tag: package-2020-03
+tag: package-2021-04
+```
+
+### Tag: package-2021-04
+
+These settings apply only when `--tag=package-2021-04` is specified on the command line.
+
+``` yaml $(tag) == 'package-2021-04'
+input-file:
+- Microsoft.Insights/stable/2015-04-01/autoscale_API.json
+- Microsoft.Insights/stable/2015-04-01/operations_API.json
+- Microsoft.Insights/stable/2016-03-01/alertRulesIncidents_API.json
+- Microsoft.Insights/stable/2016-03-01/alertRules_API.json
+- Microsoft.Insights/stable/2016-03-01/logProfiles_API.json
+- Microsoft.Insights/preview/2017-05-01-preview/diagnosticsSettings_API.json
+- Microsoft.Insights/preview/2017-05-01-preview/diagnosticsSettingsCategories_API.json
+- Microsoft.Insights/stable/2019-06-01/actionGroups_API.json
+- Microsoft.Insights/stable/2015-04-01/activityLogs_API.json
+- Microsoft.Insights/stable/2015-04-01/eventCategories_API.json
+- Microsoft.Insights/stable/2015-04-01/tenantActivityLogs_API.json
+- Microsoft.Insights/stable/2018-01-01/metricDefinitions_API.json
+- Microsoft.Insights/stable/2018-01-01/metrics_API.json
+- Microsoft.Insights/stable/2019-03-01/metricBaselines_API.json
+- Microsoft.Insights/stable/2018-03-01/metricAlert_API.json
+- Microsoft.Insights/stable/2018-04-16/scheduledQueryRule_API.json
+- Microsoft.Insights/preview/2017-12-01-preview/metricNamespaces_API.json
+- Microsoft.Insights/preview/2018-11-27-preview/vmInsightsOnboarding_API.json
+- Microsoft.Insights/preview/2019-10-17-preview/privateLinkScopes_API.json
+- Microsoft.Insights/stable/2017-04-01/activityLogAlerts_API.json
```

Comment:
Per the error message in [this issue](https://github.com/Azure/azure-sdk-for-go/issues/14704), the available api-version is listed as
```
The supported api-versions are '2014-04-01,2014-11-01,2015-04-01,2016-09-01-preview,2017-03-01-preview'
```
2017-04-01 is not one of them. Could we double confirm this is OK?

Reply:
activityLogAlerts_API.json & activityLogs_API.json are two different specs

Reply:
I am as confused as the customer right now. Is it possible that you could go to [this issue](https://github.com/Azure/azure-sdk-for-go/issues/14704) and explain this?

Reply:
Using rest API directly (https://docs.microsoft.com/en-us/rest/api/monitor/activity-logs/list) I was able to see the error message.
ARMs manifest I do see the versions, but no spec file. That why GO client do not reflect the latest API.
I'll contact the RP owner to understand why there's no spec file(s)